backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/Article.java

```diff
@@ -0,0 +1,20 @@
+package com.beetleblog.app.domains;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Date;
+
+@Getter
+@Setter
+@ToString
```

Comment:
We need to have other **Lombok** annotations here - `@EqualsAndHashCode`, `@NoArgsConstructor` and `AllArgsConstructor`,

Reply:
fixed